ssg/boolean_expression.py

```diff
@@ -112,6 +112,10 @@ def version_definitions(self):
     def name(self):
         return self.requirement.project_name
 
+    @staticmethod
+    def _is_parametrized(ref):
```

Comment:
So now the method shouldn't be private, as you intend it to be used by other callers, and `ref` doesn't give a hint about what is expected as parameter. In fact, the naming and usage `Symbol.is_parametrized` is misleading, because it's not the symbol and its parametrization that you are evaluating - instead, you are looking at a string that represents a platform unit. So please give a thought about a name of the method and of the argument as well.